### ISSUE TITLE
High battery drain under active-use

### ISSUE THREAD

**Issue by jtc42** [amirzaidi/Launcher3#216]
## Description
Launcher seems to be using a disproportionately high amount of battery life under (what I would consider to be) normal operation. See screenshot:
https://goo.gl/photos/CGi2ZMdg1Redj1p27

I'm wondering if this is something to do with debugging perhaps, as I've built this from source and may have missed some configuration?

## Expected Behavior
For 9m active use, 14% battery seems excessive, so I'd expect significantly lower.

## Actual Behavior
With no background usage, and only 9m screen-on time, launcher3 drains 14% of battery on Nexus 5X

## Environment
* Device: Nexus 5X 32GB
* Android version: 8.0
* Launcher version: Build from source, 2017 09 04
* Rom: Stock

## Logcat
N/A

**Comment by abdullahibneat** [comment]
Nexus 5X, Android Oreo, RPL 2.1 (APK from release section), phone last charged 6 hours ago, 0% battery use: https://photos.app.goo.gl/IYqrKWN2GrjdLmW12

**Comment by jtc42** [comment]
Okay strange, perhaps it's related to one of the new features not yet in the release APKs, like app suggestions. Thanks!

Edit: I've rebuilt the APK from the 2.1 release source and I'm giving that a go. Will report back or close issue based on the results.

**Comment by jtc42** [comment]
I'm still getting this issue on 2.1 when compiled from source, see https://goo.gl/photos/ESpTAkFLd4sV7kyt7

I'm happy to try with the pre-built APK, but I don't see what would be causing this just because I'm building it myself.

**Comment by abdullahibneat** [comment]
Is there any particular reason you're compiling from source?

**Comment by jtc42** [comment]
Partly because I was interested to see how launchers differed from other apps, and partly to tweak some of the UI elements like colours (only individual values in the XML files, so nothing I'd expect to hinder battery life). Nothing important.

I'm trying with the normal pre-compiled APK now to see how it goes, but it strikes me as strange that compiling the same code on two different machines would give such vastly different battery usage. That's the main reason I've got this flagged as an issue.

Update: Tried pre-compiled and works fine. Recompiled from source and I think the issue was that I was previously using a version compiled with the debugger. When compiled using normal build issue was no longer present.

Thanks for your help!
